### PR TITLE
Add Polygonal Airspace at ENGM

### PR DESCRIPTION
**Pull request by Fechulo**
-Added the Oslo TMA
-Removed restricted area D106 (it's outside the Oslo TMA)

### CODE REVIEW

**Comment by erikquinn** [comment]
Can't quite find it, but something in here is breaking ENGM.

**Comment by Fechulo** [comment]
mmh, I'll check

**Comment by Fechulo** [comment]
It's working for me, is it just not loading?

**Comment by erikquinn** [comment]
Nah, it works. It just dies when it tries to run `vsub()`, which indicates to me that it's probably something in the airspace boundary.

image

**Comment by erikquinn** [comment]
If you timewarp by like 100 or so, you get it in a few seconds.

**Comment by Fechulo** [comment]
I was testing it on an old version, let me try again.

**Comment by Fechulo** [comment]
Found it! 
I accidentally removed fix "_ENGM103060", which was being used for the ESEBA STAR.
image

I'm not sure what this fix is for, which is probably the reason why I deleted it.

**Comment by erikquinn** [comment]
All better :) Thanks!
